static/sw.js

```diff
@@ -1 +1,17 @@
 // THIS FILE SHOULD NOT BE VERSION CONTROLLED
+
+// https://github.com/NekR/self-destroying-sw
```

Comment:
For my own understanding and reference, is there a reason to remove the service workers?

Reply:
Sorry, this is not from me directly as far as I can recall. When I initially cloned the project, I had to delete the yarn.lock file and run yarn install to get it working on my machine. I'm not sure if this has something to do with that.

Reply:
ah, yea not sure. I know that if you run the build script locally, it updates this file as well. When that happens I just do a git checkout of the file and it will stash the changes to restore the defaults.

Reply:
If you didn't intentionally add this file as part of the shareWidget, I would remove it too. Not sure if this app uses service workers, but if a web app has service workers, sometimes it's part of a pwa (progressive web app) setup. Where the service workers help with things like caching the app for speed boost, offline viewing, etc.

Reply:
Will remove, thanks for the explanation.